README.md

```diff
@@ -1,7 +1,27 @@
-# Tauri + SvelteKit + TypeScript
 
-This template should help get you started developing with Tauri, SvelteKit and TypeScript in Vite.
+# MTracks
+
+MTracks is an app designed to make playing of MultiTracks easier and better through a simple UI and advanced features.
+
+
+## Features
+
+- Light/dark mode toggle
+- Live previews
+- Fullscreen mode
+- Cross platform
+- Local MultiTracks
+- Free to use
+
+## Authors
```

Comment:
Please Remove authers from the readme, it is already in the git. move this to a authors file